src/server/worldserver/worldserver.conf.dist

```diff
@@ -2294,9 +2294,16 @@ Rate.XP.Pet       = 1
 #    Rate.XP.BattlegroundKill
 #        Description: Experience rate for honorable kills in battlegrounds. Not affected by Rate.XP.Kill
 #                     Only works if Battleground.GiveXPForKills = 1
+#                     Can be overwritten for each BG by setting a value different than -1 on Rate.XP.BattlegroundKill???, where ??? is the name of the BG.
 #        Default:     1
 
-Rate.XP.BattlegroundKill = 1
+Rate.XP.BattlegroundKill     = 1
+Rate.XP.BattlegroundKillAV   = -1
+Rate.XP.BattlegroundKillWSG  = -1
+Rate.XP.BattlegroundKillAB   = -1
+Rate.XP.BattlegroundKillEOTS = -1
+Rate.XP.BattlegroundKillSOTA = -1
+Rate.XP.BattlegroundKillIC   = -1
```

Comment:
Why not set all to 1? 

1 * default will still be default

Reply:
да, кажется лучше

Reply:
Определённо

Reply:
@Kitzunu I developed this way so you can set them to 1 if BattlegroundKill = X where X > 1, so you can always override it. I tried with 0, but 0 and floats don' get along too well.

Reply:
I'd propose dropping the general XP rate and just setting all rates to 1.0f as we do the same on several other occasions.

Reply:
Because a general xp rate is not needed when you can twek them anyways

Reply:
Yeah, I can do that, I didn't want to because of repentance and to leave the config opt as it is for retro-compatibility.

Reply:
I think it's better like it currently the PR is

Reply:
I disagree:
1) the code looks so much cleaner with the 1x as default and it's more intuitive as that's how every other rates setting is currently implemented
2) the override in this scenario becomes obsolete because each setting can be changed individually, so if one wishes to keep them as they used to be they need to do nothing

Reply:
Fair enough, I'll change it then